actfw_core/system.py

```diff
@@ -0,0 +1,85 @@
+import os
+from typing import Optional, cast
+
+
+def _get_env_str(name: str) -> str:
+    return cast(str, os.environ.get(name))
```

Comment:
The returned type should be `Optional[str]` .

```
>>> cast(str, None) is None
True
```

Reply:
This is intended behavior